src/multivariate/mvtdist.jl

```diff
@@ -103,7 +103,6 @@ logdet_cov(d::GenericMvTDist) = d.df>2 ? logdet((d.df/(d.df-2))*d.Σ) : NaN
 
 params(d::GenericMvTDist) = (d.df, d.μ, d.Σ)
 @inline partype(d::GenericMvTDist{T}) where {T} = T
-eltype(::GenericMvTDist{T}) where {T} = T
```

Comment:
Why remove the eltype here?

Reply:
Because it's handled by the default case now. In `src/common.jl`:
```julia
Base.eltype(::Sampleable{F, <: ValueSupport{N}}) where {F, N} = N
```

Reply:
Got it thank you